client/wfnews-war/src/main/resources/spring/security/spring-config.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, we would get rid of the XML configuration and use class based configuration with Spring 5